packages/components/src/components/divider/divider.e2e-test.ts

```diff
@@ -1,65 +1,208 @@
+      await dividerStickerSheet.createMarkupWithCombination({ variant: DIVIDER_VARIANT,
+        'arrow-direction':
+      DIRECTIONS,
+        'button-position': DIRECTIONS });
+
+      await dividerStickerSheet.mountStickerSheet();
+      const container = dividerStickerSheet.getWrapperContainer();
+      await container.evaluate((wrapper) => {
+        const modifiedWrapper = wrapper;
+        modifiedWrapper.style.padding = '1.25rem';
+      });
+      await container.evaluate((el) => {
+        const rowWrappers = el.querySelectorAll('.componentRowWrapper') as NodeListOf<HTMLElement>;
+        Array.from(rowWrappers).forEach((rowWrapper) => {
+          const modifiedRowWrapper = rowWrapper;
+          modifiedRowWrapper.style.gap = '1.5rem';
+          modifiedRowWrapper.style.marginBottom = '3.5rem';
+        });
+      });
+      await test.step('matches screenshot of element', async () => {
+        await componentsPage.visualRegression.takeScreenshot('mdc-divider-horizontal', { element: container });
       });
 
-      // add additional tests here, like tabbing through several parts of the component
+      /**
+     * ACCESSIBILITY
+     */
+      await test.step('accessibility', async () => {
+        await componentsPage.accessibility.checkForA11yViolations('mdc-divider-horizontal');
+      });
     });
 
-    await test.step('keyboard', async () => {
-      await test.step('component should fire callback x when pressing y', async () => {
-        // TODO: add test here
+    await test.step('Vertical', async () => {
+      // for primary type
+      dividerStickerSheet.setAttributes({ orientation: DIVIDER_ORIENTATION.VERTICAL });
+      dividerStickerSheet.setChildren(NoChildren);
+      await dividerStickerSheet.createMarkupWithCombination({ variant: DIVIDER_VARIANT });
+
+      // for grabber type
+      dividerStickerSheet.setAttributes({ orientation: DIVIDER_ORIENTATION.VERTICAL });
+      dividerStickerSheet.setChildren(Grabber);
+      await dividerStickerSheet.createMarkupWithCombination({ variant: DIVIDER_VARIANT,
+        'arrow-direction':
+      DIRECTIONS,
+        'button-position': DIRECTIONS });
+
+      await dividerStickerSheet.mountStickerSheet();
+      const container = dividerStickerSheet.getWrapperContainer();
+      await container.evaluate((wrapper) => {
+        const modifiedWrapper = wrapper;
+        modifiedWrapper.style.display = 'flex';
+        modifiedWrapper.style.gap = '3.5rem';
+        modifiedWrapper.style.height = '90%';
+        modifiedWrapper.style.padding = '1.25rem';
+      });
+
+      await container.evaluate((el) => {
+        const rowWrappers = el.querySelectorAll('.componentRowWrapper') as NodeListOf<HTMLElement>;
+        Array.from(rowWrappers).forEach((rowWrapper) => {
+          const modifiedRowWrapper = rowWrapper;
+          modifiedRowWrapper.style.gap = '3.5rem';
+          modifiedRowWrapper.style.marginBottom = '3.5rem';
+        });
+      });
```

Comment:
```suggestion
  await container.evaluate((wrapper) => {
        const modifiedWrapper = wrapper;
        modifiedWrapper.style.display = 'flex';
        modifiedWrapper.style.gap = '3.5rem';
        modifiedWrapper.style.height = '90%';
        modifiedWrapper.style.padding = '1.25rem';

        const rowWrappers = wrapper.querySelectorAll('.componentRowWrapper') as NodeListOf<HTMLElement>;
        Array.from(rowWrappers).forEach((rowWrapper) => {
          const modifiedRowWrapper = rowWrapper;
          modifiedRowWrapper.style.gap = '3.5rem';
          modifiedRowWrapper.style.marginBottom = '3.5rem';
        });
      });
```
Both of them can be combined in a single block of evaluate.